week2/hackyourtemperature/app.js

```diff
@@ -0,0 +1,40 @@
+import fetch from 'node-fetch';
+import express from 'express';
+import { keys } from './sources/keys.js';
+
+const app = express();
+
+app.use(express.json());
+app.use(express.urlencoded({ extended: false }));
+
+app.get('/', (req, res) => {
+  res.send('hello from backend to frontend!');
+});
+
+let temperature;
+app.post('/weather', async (req, res) => {
+  const cityName = req.body.cityName;
```

Comment:
Since the variable name is the same as the value you are trying to get from the object, you can also simply destructure the value

```suggestion
  const { cityName } = req.body;
```

Reply:
Thanks! I've updated my code according to your feedback.